scripts/gen_dlaf_strong-gpu.py

```diff
@@ -96,20 +136,21 @@ def createAndSubmitRun(run_dir, nodes_arr, dtype, **kwargs):
     run.add(
         mp.bt_band2trid,
         params={"rpn": rpn, "m_sz": m_szs, "mb_sz": mb_szs, "band": 128, "n_sz": None},
-        **full_kwargs,
+        **fullbt_kwargs,
     )
     run.add(
         mp.bt_red2band,
         params={"rpn": rpn, "m_sz": m_szs, "mb_sz": mb_szs, "band": 128, "n_sz": None},
-        **full_kwargs,
+        **fullbt_kwargs,
     )
+    # TODO: Partial trsm for Cholesky
```

Comment:
To do?

Reply:
Well spotted, thanks. So far I only tested the standard eigensolver, so this was not yet needed (but the miniapp has already been updated).